cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/IndexManager.java

```diff
@@ -385,23 +361,27 @@ protected Datastore getDatastore() {
     }
 
     protected ExecutorService getQueue() {
-        return queue;
+        return null;
+    }
+
+    protected SQLDatabaseQueue getDatabaseQueue() {
```

Comment:
I don't like adding the method just for the tests, that's probably how we ended up with `getQueue()` and `getDatabase()` here in the first place.
Admittedly I'm not a huge fan of reflectively obtaining it in the base class of the tests, but I think that's preferable to adding a method.